private/scripts/helpers.sh

```diff
@@ -54,6 +68,10 @@ function get_info() {
 
   # Unset the variables if we're doing this a second time.
   if [ "$is_restarted" -eq 1 ]; then
+    if [ "$is_ci" -eq 1 ]; then
```

Comment:
`-eq` might be unhappy if compared to `""`? Default to 0 up on line 23, or test with -n/-z first or just use [[ == ]] instead.